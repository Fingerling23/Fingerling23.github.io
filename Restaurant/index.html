<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
<title>Restaurant</title>
<link href="https://fonts.googleapis.com/css?family=Cormorant+Garamond" rel="stylesheet">	
<link rel="stylesheet" href="css/style.css">
<link href="https://fonts.googleapis.com/css?family=Bad+Script" rel="stylesheet">
</head>
<body>
<div class="wrap">
	<head>
		<nav>
			<ul class="top-nav">
				<li><img src="images/nav-logo.png" alt=""></li>
				<li><a href="#">О ресторане</a></li>
				<li><a href="#">Меню</a></li>
				<li><a href="#">Галерея</a></li>
				<li><a href="#">Новости</a></li>
				<li><a href="#">Вакансии</a></li>
				<li><a href="#">Контакты</a></li>
				<a class="social fb" href="#"></a>
				<a class="social vk" href="#"></a>
				<a class="social inst" href="#"></a>
			</ul>
		</nav>

		<div class="back-logo">
			<img src="images/fish-logo.png" alt="">
			<div class="heading">
				<h1>ВАШ ЗАГОЛОВОК</h1> 
				<p>Добро пожаловать!</p>
				<span>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Sunt, molestias?</span>
			</div>
		</div>
	</head>

	<section class="section-1">
		
		<div>
			<h2>О РЕСТОРАНЕ</h2>
			<span>Добро пожаловать!</span>
			<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Praesentium voluptas, vero omnis assumenda expedita perferendis culpa! Vero porro quaerat similique repudiandae. Temporibus repellendus, excepturi numquam voluptatibus voluptates doloremque blanditiis architecto vitae! Ducimus rerum, obcaecati saepe eligendi, fugiat animi inventore enim tempora dicta incidunt in nulla maiores, possimus ex iusto iste, ea explicabo nemo distinctio. A expedita magni sequi, ut quisquam hic veniam, maxime, fugiat perspiciatis placeat iusto quidem laborum obcaecati magnam iste, adipisci. Adipisci magnam odit dolorem. Eius officiis facere tempora id, quaerat est ipsum laborum perspiciatis a delectus veniam? Quia maxime eveniet magnam molestias at culpa facilis molestiae ea!</p>
		</div>
	</section>

	<section class="section-2">
		<div>
			<h2>МЕНЮ</h2>
				<div class="subtitle">
					<span>Вкусные предложения!</span>
				</div>
			<div class="items">
			<figure>
				<img src="images/breakfast-logo.png" alt="">
				<figcaption><h3>ЗАВТРАКИ</h3><span><br>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Itaque sit accusantium, voluptatibus dolorum perferendis qui?</span>
					<a href="#">Подробнее<img src="images/nav-direction-r.png" alt=""></a>
				</figcaption>
			</figure>
			<figure>
				<img src="images/fish-dish.png" alt="">
				<figcaption><h3>БЛЮДА ИЗ РЫБЫ</h3><span><br>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Explicabo earum, veritatis fugit, cupiditate aspernatur minima!</span>
					<a href="#">Подробнее<img src="images/nav-direction-r.png" alt=""></a>
				</figcaption>
			</figure>
			</div>

			<div class="items">
			<figure>
				<img src="images/fried-dish.png" alt="">
				<figcaption><h3>БЛЮДА ИЗ РЫБЫ</h3><span><br>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Explicabo earum, veritatis fugit, cupiditate aspernatur minima!</span>
					<a href="#">Подробнее<img src="images/nav-direction-r.png" alt=""></a>
				</figcaption>
			</figure>
			<figure>
				<img src="images/cold-snaks.png" alt="">
				<figcaption><h3>ХОЛОДНЫЕ ЗАКУСКИ</h3><span><br>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Unde accusamus, quibusdam ullam fugiat vero soluta.</span>
					<a href="#">Подробнее<img src="images/nav-direction-r.png" alt=""></a>
				</figcaption>
			</figure>
			</div>
		</div>
	</section>

	<section class="section-3">
		
		<div>
			<h2>НОВОСТИ</h2>
				<div class="subtitle">
					<img src="images/nav-direction-l.png" alt="">
					<span>Узнайте больше вместе с нами!</span>
					<img src="images/nav-direction-r.png" alt="">
				</div>
			<div class="items-2">
			<figure>
				<img src="images/first-logo.png" alt="">
				<figcaption><span><br><h3>Lorem ipsum dolor sit amet.</h3>Lorem ipsum dolor sit amet, consectetur adipisicing.</span>
					<a href="#">Подробнее</a>
				</figcaption>
			</figure>
			<figure>
				<img src="images/second-logo.png" alt="">
				<figcaption><span><br><h3>Lorem ipsum dolor sit amet.</h3>Lorem ipsum dolor sit amet, consectetur adipisicing.</span>
					<a href="#">Подробнее</a>
				</figcaption>
			</figure>
			</div>

			<div class="items-2">
			<figure>
				<img src="images/third-logo.png" alt="">
				<figcaption><span><br><h3>Lorem ipsum dolor sit amet.</h3>Lorem ipsum dolor sit amet, consectetur adipisicing.</span>
					<a href="#">Подробнее</a>
				</figcaption>
			</figure>
			<figure>
				<img src="images/fourth-logo.png" alt="">
				<figcaption><span><br><h3>Lorem ipsum dolor sit amet.</h3>Lorem ipsum dolor sit amet, consectetur adipisicing.</span>
					<a href="#">Подробнее</a>
				</figcaption>
			</figure>
			</div>
		</div>
	</section>

	<section class="section-4">
		<h2>КОНТАКТЫ</h2>
		<div class="contacts">
			<img src="images/place-icon.png" alt=""><p>Город. Улица n 10</p><br>
			<img src="images/tel-icon.png" alt=""><p>+0 (000) 000 00 00</p><br>
			<img src="images/mail-icon.png" alt=""><p>info@pochta.ru</p><br>
			<img src="images/clock-icon.png" alt=""><p>Пн-Пт с 9:00 до 18:00</p>
		</div>
	</section>
	
	</div> <!-- wrap -->

	<footer>
		<nav>
			<ul class="bot-nav">
				<li><a href="#">О ресторане</a></li>
				<li><a href="#">Меню</a></li>
				<li><a href="#">Галерея</a></li>
				<li><a href="#">Новости</a></li>
				<li><a href="#">Вакансии</a></li>
				<li><a href="#">Контакты</a></li>
					<p>&copy; 2018 Сайт создан by Tim Panov</p>
			</ul>
		</nav>
	</footer>
</body>
</html>

