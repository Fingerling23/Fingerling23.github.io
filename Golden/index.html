<!doctype html>
<html lang="en">

<head>
    <!-- Required meta tags -->
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <!-- Bootstrap CSS -->
    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css" integrity="sha384-MCw98/SFnGE8fJT3GXwEOngsV7Zt27NXFoaoApmYm81iuXoPkFOJwJ8ERdknLPMO" crossorigin="anonymous">
    <link rel="stylesheet" href="css/style.css">
    <link rel="stylesheet" href="css/normalize.css">
    <link href="https://fonts.googleapis.com/css?family=KoHo" rel="stylesheet">
    <title>Golden</title>
</head>

<body>
    <header>
        <div class="top_line">
            <div class="container">
                <div class="row">
                    <div class="col-md-2">
                        <div class="gold_logo">
                            <a href="#"><img src="img/golden_logo.png" alt="Golden"></a>
                        </div>
                    </div>
                    <div class="col-md-10">
                        <nav class="top_mnu navbar navbar-light navbar-1 white">
                            <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent15" aria-controls="navbarSupportedContent15" aria-expanded="false" aria-label="Toggle navigation"><span class="navbar-toggler-icon"></span></button>
                            <!-- Collapsible content -->
                            <div class="collapse navbar-collapse" id="navbarSupportedContent15">
                                <ul class="menu navbar-nav mr-auto col-sm-12">
                                    <li class="nav-item">
                                        <a href="#">Home</a>
                                    </li>
                                    <li>
                                        <a href="#">Services</a>
                                    </li>
                                    <li>
                                        <a href="#">Portfolio</a>
                                    </li>
                                    <li>
                                        <a href="#">About</a>
                                    </li>
                                    <li>
                                        <a href="#">Contact</a>
                                    </li>
                                </ul>
                            </div>
                        </nav>
                    </div>
                </div>
            </div>
        </div>
        <div class="titles">
            <div class="container">
                <div class="row">
                    <div class="col-md-12 col-sm-12">
                        <p>Welcome To Our Studio!</p>
                        <h1>It's nice to meet you</h1>
                    </div>
                </div>
            </div>
        </div>
        <div class="head_butt">
            <div class="container">
                <div class="row">
                    <div class="col-md-12 col-sm-12">
                        <div class="yellow_but">
                            <a href="#">Tell me more</a>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </header>
    <section class="servi">
        <div class="container">
            <div class="row">
                <div class="col-md-12 col-sm-12">
                    <div class="title_text">
                        <h2>Services</h2>
                        <p>Proin iaculis purus consequat sem cure.</p>
                    </div>
                </div>
            </div>
        </div>
    </section>
    <section>
        <div class="serv_items">
            <div class="container">
                <div class="row">
                    <div class="col-md-4 col-sm-12">
                        <div class="content">
                            <a href="#"><img src="img/basket_icon.png" alt=""></a>
                            <h3>E-Commerce</h3>
                            <p>Proin iaculis purus consequat sem cure
                                digni ssim. Donec porttitora entum suscipit
                                aenean rhoncus posuere odio in tincidunt.</p>
                        </div>
                    </div>
                    <div class="col-md-4 col-sm-12">
                        <div class="content">
                            <a href="#"><img src="img/laptop_icon.png" alt=""></a>
                            <h3>Responsive Web</h3>
                            <p>Proin iaculis purus consequat sem cure
                                digni ssim. Donec porttitora entum suscipit
                                aenean rhoncus posuere odio in tincidunt.</p>
                        </div>
                    </div>
                    <div class="col-md-4  col-sm-12">
                        <div class="content">
                            <a href="#"><img src="img/locker_icon.png" alt=""></a>
                            <h3>Web Security</h3>
                            <p>Proin iaculis purus consequat sem cure
                                digni ssim. Donec porttitora entum suscipit
                                aenean rhoncus posuere odio in tincidunt.</p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>
    <section>
        <div class="our_port">
            <div class="container">
                <div class="pic_items">
                    <div class="row">
                        <div class="col-md-12 col-sm-12">
                            <div class="title_text">
                                <h2>Our Portfolio</h2>
                                <p>Proin iaculis purus consequat sem cure. </p>
                            </div>
                        </div>
                        <div class="col-md-6 col-lg-4 col-sm-12">
                            <img src="img/ebony.png" alt="" class="img-fluid">
                            <div class="bg_items">
                                <h4>Ebony & Ivory</h4>
                                <p>Branding</p>
                            </div>
                        </div>
                        <div class="col-md-6 col-lg-4 col-sm-12">
                            <img src="img/smart.png" alt="" class="img-fluid">
                            <div class="bg_items">
                                <h4>Smart Stationary</h4>
                                <p>Print Design</p>
                            </div>
                        </div>
                        <div class="col-md-6 col-lg-4 col-sm-12">
                            <img src="img/clever.png" alt="" class="img-fluid">
                            <div class="bg_items">
                                <h4>Clever Poster</h4>
                                <p>Print Design</p>
                            </div>
                        </div>
                        <div class="col-md-6 col-lg-4 col-sm-12">
                            <img src="img/vinyl.png" alt="" class="img-fluid">
                            <div class="bg_items">
                                <h4>Vinyl Record</h4>
                                <p>Product Mock-Up</p>
                            </div>
                        </div>
                        <div class="col-md-6 col-lg-4 col-sm-12">
                            <img src="img/template.png" alt="" class="img-fluid">
                            <div class="bg_items">
                                <h4>Treehouse Template</h4>
                                <p>Web Design</p>
                            </div>
                        </div>
                        <div class="col-md-6 col-lg-4 col-sm-12">
                            <img src="img/burned.png" alt="" class="img-fluid">
                            <div class="bg_items">
                                <h4>Burned Logo</h4>
                                <p>Branding</p>
                            </div>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>
    <section>
        <div class="about">
            <div class="container">
                <div class="row">
                    <div class="col-md-12 col-sm-12">
                        <div class="title_text">
                            <h2>About us</h2>
                            <p>Proin iaculis purus consequat sem cure. </p>
                        </div>
                        <div class="about_titles">
                            <div class="about_items_left">
                                <div class="row">
                                    <div class="col-md-6 col-lg-4 col-sm-12">
                                        <h4>July 2010<br>Our Humble Beginnings</h4>
                                        <p>Proin iaculis purus consequat sem cure
                                            digni ssim. Donec porttitora entum suscipit
                                            aenean rhoncus posuere odio in tincidunt. Proin
                                            iaculis purus consequat sem cure digni
                                            ssim. Donec porttitora entum.</p>
                                    </div>
                                    <div class="col-md-6 col-lg-4  col-sm-12">
                                        <img src="img/july.png" alt="">
                                </div>
                                        <div class="col-lg-4  col-sm-12">
                                        </div>
                                    </div>
                                </div>
                                <div class="about_items_right">
                                    <div class="row">
                                        <div class="col-lg-4 col-sm-12">
                                        </div>
                                        <div class="col-md-6 col-lg-4 col-sm-12">
                                            <img src="img/january.png" alt="">
                                        </div>
                                            <div class="col-md-6 col-lg-4 col-sm-12">
                                                <h4>January 2011<br>Facing Startup Battles</h4>
                                                <p>Proin iaculis purus consequat sem cure
                                                    digni ssim. Donec porttitora entum suscipit
                                                    aenean rhoncus posuere odio in tincidunt. Proin
                                                    iaculis purus consequat sem cure digni
                                                    ssim. Donec porttitora entum suscipit aenean
                                                    rhoncus posuere odio in tincidunt.</p>
                                            </div>
                                        </div>
                                    </div>
                                    <div class="about_items_left">
                                        <div class="row">
                                            <div class="col-md-6 col-lg-4 col-sm-12">
                                                <h4>December 2012<br>Enter The Dark Days</h4>
                                                <p>Proin iaculis purus consequat sem cure
                                                    digni ssim. Donec porttitora entum suscipit
                                                    aenean rhoncus posuere odio in tincidunt. Proin
                                                    iaculis purus consequat sem cure.</p>
                                            </div>
                                            <div class="col-md-6 col-lg-4 col-sm-12">
                                                <img src="img/december.png" alt="">
                                            </div>
                                                <div class="col-lg-4 col-sm-12">
                                                </div>
                                            </div>
                                        </div>
                                        <div class="about_items_right">
                                            <div class="row">
                                                <div class="col-lg-4 col-sm-12">
                                                </div>
                                                <div class="col-md-6 col-lg-4 col-sm-12">
                                                    <img src="img/february.png" alt="">
                                                </div>
                                                    <div class="col-md-6 col-lg-4 col-sm-12">
                                                        <h4>February 2014<br>Our Triumph</h4>
                                                        <p>Proin iaculis purus consequat sem cure
                                                            digni ssim. Donec porttitora entum suscipit
                                                            aenean rhoncus posuere odio in tincidunt. Proin
                                                            iaculis purus consequat sem cure digni
                                                            ssim. Donec porttitora entum suscipit aenean.</p>
                                                    </div>
                                                </div>
                                            </div>
                                            <div class="row">
                                                <div class="col-md-4 col-lg-4 col-sm-12"></div>
                                                <div class="col-md-12 col-lg-4 col-sm-12">
                                                    <div class="story">
                                                        <h4>Our<br>story<br>continues<br>...</h4>
                                                    </div>
                                                </div>
                                                <div class="col-lg-4 col-sm-12"></div>
                                            </div>
                                        </div>
                                    </div>
                                </div>
                            </div>
                        </div>
    </section>
    <section>
        <div class="team">
            <div class="container">
                <div class="row">
                    <div class="col-md-12 col-sm-12">
                        <div class="title_text">
                            <h2>Our amazing team</h2>
                            <p>Proin iaculis purus consequat sem cure. </p>
                        </div>
                    </div>
                </div>
                <div class="team_members">
                    <div class="row">
                        <div class="col-md-4 col-sm-12">
                            <div class="team_items">
                                <img src="img/photo.png" alt="photo">
                                <h5>Kimberly Thompson</h5>
                                <p>Marketer</p>
                                <a href="#"><img src="img/tweeter_icon.png" alt=""></a>
                                <a href="#"><img src="img/facebook_icon.png" alt=""></a>
                                <a href="#"><img src="img/google_icon.png" alt=""></a>
                            </div>
                        </div>
                        <div class="col-md-4 col-sm-12">
                            <div class="team_items">
                                <img src="img/photo.png" alt="photo">
                                <h5>Rico Massimo</h5>
                                <p>Coder</p>
                                <a href="#"><img src="img/tweeter_icon.png" alt=""></a>
                                <a href="#"><img src="img/facebook_icon.png" alt=""></a>
                                <a href="#"><img src="img/google_icon.png" alt=""></a>
                            </div>
                        </div>
                        <div class="col-md-4 col-sm-12">
                            <div class="team_items">
                                <img src="img/photo.png" alt="photo">
                                <h5>Uku Mason</h5>
                                <p>Graphic Designer</p>
                                <a href="#"><img src="img/tweeter_icon.png" alt=""></a>
                                <a href="#"><img src="img/facebook_icon.png" alt=""></a>
                                <a href="#"><img src="img/google_icon.png" alt=""></a>
                            </div>
                        </div>
                    </div>
                </div>
                <div class="row">
                    <div class="col-md-2 col-sm-12">
                    </div>
                    <div class="col-md-8 col-sm-12">
                        <div class="under_team">
                            <p>Proin iaculis purus consequat sem cure digni ssim donec porttitora entum suscipit
                                aenean rhoncus posuere odio in tincidunt proin iaculis.</p>
                        </div>
                    </div>
                    <div class="col-md-2 col-sm-12">
                    </div>
                </div>
            </div>
        </div>
    </section>
    <section>
        <div class="cms_systems">
            <div class="container">
                <div class="row">
                    <div class="col-md-12 col-sm-12">
                        <div class="cms_position">
                            <a href="#"><img src="img/evanto.png" alt=""></a>
                            <a href="#"><img src="img/wordpress.png" alt=""></a>
                            <a href="#"><img src="img/aetuts.png" alt=""></a>
                            <a href="#"><img src="img/microlancer.png" alt=""></a>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>
    <footer>
        <section>
            <div class="contact">
                <div class="container">
                    <div class="row">
                        <div class="col-md-12 col-sm-12">
                            <div class="title_text">
                                <h2>Contact us</h2>
                                <p>Proin iaculis purus consequat sem cure. </p>
                            </div>
                        </div>
                    </div>
                    <div class="form">
                        <div class="row">
                            <div class="col-md-6 col-sm-12">
                                <form id="cell">
                                    <input type="text" placeholder="Your name" required>
                                    <input type="text" placeholder="Your email" required>
                                    <input type="text" placeholder="Subject">
                                </form>
                            </div>
                            <div class="col-md-6 col-sm-12">
                                <form>
                                    <input type="text" placeholder="Your message" class="message">
                                </form>
                            </div>
                        </div>
                    </div>
                    <div class="head_butt">
                        <div class="container">
                            <div class="row">
                                <div class="col-md-12">
                                    <div class="yellow_but">
                                        <a href="#">Send message</a>
                                    </div>
                                </div>
                            </div>
                        </div>
                    </div>
                </div>
            </div>
        </section>
        <section>
            <div class="copyright">
                <div class="container">
                    <div class="row">
                        <div class="col-md-4 col-sm-12 copy">
                            <p>&copy;Copyright 2014 FreebiesXpress.com</p>
                        </div>
                        <div class="col-md-4 col-sm-12">
                            <div class="footer_links">
                                <a href="#"><img src="img/tweeter_footer_icon.png" alt=""></a>
                                <a href="#"><img src="img/facebook_footer_icon.png" alt=""></a>
                                <a href="#"><img src="img/google_footer_icon.png" alt=""></a>
                            </div>
                        </div>
                        <div class="col-md-4 col-sm-12"></div>
                    </div>
                </div>
            </div>
        </section>
    </footer>
    <!-- Optional JavaScript -->
    <!-- jQuery first, then Popper.js, then Bootstrap JS -->
    <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js" integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.3/umd/popper.min.js" integrity="sha384-ZMP7rVo3mIykV+2+9J3UJ46jBk0WLaUAdn689aCwoqbBJiSnjAK/l8WvCWPIPm49" crossorigin="anonymous"></script>
    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/js/bootstrap.min.js" integrity="sha384-ChfqqxuZUCnJSK3+MXmPNIyE6ZbWh2IMqE241rYiqJxyMiZ6OW/JmZQ5stwEULTy" crossorigin="anonymous"></script>
</body>

</html>